styleguide/components/31-molecules/contact-block/contact-block.twig

```diff
@@ -1,4 +1,4 @@
-<div class="contact-block box-left">
+<div class="contact-block box-left {% if tel %}tel{% endif %}{% if email %}mail{% endif %}">
```

Comment:
I think your "tel" and "email" classes aren't gonna be separated by a space here?

Reply:
well... `tel` and `email` never occur at the same time, so it shouldn't be a problem 🙂